docs/modules/slamdunk.md

```diff
@@ -0,0 +1,24 @@
+---
+Name: Slamdunk
+URL: https://github.com/t-neumann/slamdunk
+Description: >
+    Slamdunk is a tool to analyze SLAM-Seq data.
+---
+
+Blablabla
+[Cutadapt](https://code.google.com/p/cutadapt/),
```

Comment:
Yup, this needs to be written. Would be nice to have a short primer as to what SLAM-Seq actually is..! A quick google doesn't reveal much.